src/main.cpp

```diff
@@ -1111,7 +1114,7 @@ QString UsageArguments ( char** argv )
            "  -F, --fastupdate        use 64 samples frame size mode\n"
            "  -l, --log               enable logging, set file name\n"
            "  -L, --licence           show an agreement window before users can connect\n"
-           "  -m, --htmlstatus        enable HTML status file, set file name\n"
+           "  -m, --htmlstatus        Deprecated. Use JSON-RPC instead\n"
```

Comment:
```suggestion
           "  -m, --htmlstatus        deprecated, please use JSON-RPC instead\n"
```
Just noticed the inconsistency.

Reply:
Done.